src/core/sys/windows/winnt.d

```diff
@@ -383,43 +384,43 @@ enum : WORD {
     SECURITY_MANDATORY_PROTECTED_PROCESS_RID = 0x5000,
     SECURITY_MANDATORY_MAXIMUM_USER_RID      = SECURITY_MANDATORY_SYSTEM_RID
 }
-
+package enum wchar_t[] TEXT(string a) = cast(wchar_t[])a;
```

Comment:
Shouldn't this keep immutable/const?

Maybe better as a regular CTFE function to avoid the template bloat in memory and object file?

Reply:
You can cast a literal to wstring at compile time fine, you'd have to decode it if you wanted that as a function wouldn't you? Anyway feel free to make any changes you think are appropriate. I don't do windows stuff, I'm just trying to make this pass.